src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form-role.model.ts

```diff
@@ -4,11 +4,12 @@ export class UserFormRoleModel implements SelectOption {
   name: string;
   description: string;
   selected = false;
-  scopes_permissions: object;
-  enabled = true;
-
-  constructor(name: string, description: string) {
+  scopes_permissions: object = {};
```

Comment:
but is this required?
```suggestion
  scopes_permissions?: object;
```

Reply:
scopes_permissions?: object;  suggested change are done